### ISSUE TITLE
Required method for waiting of empty values and regular expressions in response

### ISSUE THREAD

**Issue by AleksandrIakhnev** [VolodymyrLavrenchuk/PySlim#20]
I have a response like `{'metadata': 'members', 'value':  []}` Any idea how can I wait for an empty array with PySlim?  Also would be good to have a method for waiting for regex - for example for waiting for object guid in objects list.

**Comment by mikeplavsky** [comment]
So waiting for attribute `value` being equal `[]` does not work ?

**Comment by AleksandrIakhnev** [comment]
Nope, because there should be parsing of string with value `[]`to value of empty list. Maybe here https://github.com/VolodymyrLavrenchuk/PySlim/blob/master/http_calls.py#L224  we can compare string representations instead of casting to specific types.

**Comment by mikeplavsky** [comment]
I would add an additional method, using the same technique `self.wait(wait_sec, retries, func, attr=attr, url=url)` but with different `func`

There are already couple: `waitSecondTimesUrlResponseAttributeHasValue` and `waitSecondTimesUrlResponseAttributeNotZero`

**Comment by AleksandrIakhnev** [comment]
Ok, I'll make a pull request
